src/templates/root.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from "react";
+import PageLayout from "../components/PageLayout";
+import Banner from "../components/Banner";
+import DirectoryRootGrid from "../components/DirectoryRootGrid";
+import { directoryRootGridFields } from "../components/DirectoryRootGrid";
+import Favicon from "../assets/images/yext-favicon.ico";
+import "../index.css";
+import {
+  Template,
+  GetPath,
+  GetRedirects,
+  TemplateConfig,
+  TemplateProps,
+  TemplateRenderProps,
+  TransformProps,
+  GetHeadConfig,
+  HeadConfig,
+} from "@yext/pages";
+
+/**
+ * Required when Knowledge Graph data is used for a template.
+ */
+export const config: TemplateConfig = {
+  stream: {
+    $id: "root-stream",
+    filter: {
+      entityTypes: ["ce_root"],
+    },
+    fields: [
+      "id",
+      "uid",
+      "meta",
+      "name",
+      "slug",
+      ...directoryRootGridFields,
```

Comment:
I think we should explicitly type out the fields here rather that use the list of fields imported from DirectoryRootGrid